src/components/cc-logs-instances/cc-logs-instances.stories.js

```diff
@@ -0,0 +1,324 @@
+import './cc-logs-instances.js';
+import { makeStory } from '../../stories/lib/make-story.js';
+import { enhanceStoriesNames } from '../../stories/lib/story-names.js';
+
+/**
+ * @typedef {import('./cc-logs-instances.types.d.ts').Instance} Instance
```

Comment:
question: just curious does that help the IDE doing that here?

Reply:
yep